src/controllers/userController.js

```diff
@@ -56,14 +56,20 @@ userController.post('/register', async (req, res) => {
       role: req.body.role ? req.body.role : 'regular',
       region: req.body.region ? req.body.region : '',
       phone: req.body.phone ? req.body.phone : '',
+      isDeleted: false,
       hashedPassword: await bcrypt.hash(req.body.password, 10),
     };
+    if (userData.role == 'regular') {
+      userData.isApproved = true;
```

Comment:
And what about admins?

Reply:
the default value is false and will be waiting approval.